todo/tests/integration/base_mongo_test.py

```
@@ -0,0 +1,27 @@
+from django.test import TransactionTestCase, override_settings
+from pymongo import MongoClient
+from todo.tests.testcontainers.shared_mongo import get_shared_mongo_container
+from todo_project.db.config import DatabaseManager
+
+@override_settings(DB_NAME="testdb")
+class BaseMongoTestCase(TransactionTestCase):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)